addons/crm_iap_lead_enrich/models/crm_lead.py

```diff
@@ -53,7 +62,7 @@ def iap_enrich(self, from_cron=False):
                         {'lead_ids': tuple(leads.ids)}, log_exceptions=False)
                     for lead in leads:
                         # If lead is lost, active == False, but is anyway removed from the search in the cron.
-                        if lead.probability == 100 or lead.iap_enrich_done:
+                        if lead.probability == 100 or lead.iap_enrich_done and not lead.active:
```

Comment:
Not sure I understand why this change is related to your commit here?

Reply:
Hello @awa-odoo 

Actually, we don't want to show the Bus notification if a user is importing the file.
Because it may trigger this notification multiple times on the user screen.

Thanks

Reply:
We don't want to enrich the archived lead.